### ISSUE TITLE
Ordinary RMarkdown files in a Quarto project don't have a `Knit` button

### ISSUE THREAD

**Issue by cmarquardt** [rstudio/rstudio#11003]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2022.02.01 - Build 461
    Quarto Version  : 0.9.258
    OS Version      : macOS Catalina (10.15.7)
    R Version       : 4.1.3

### Steps to reproduce the problem

Inside a Quarto project, create an RMarkdown file.

### Describe the problem in detail

The RStudio UI shows Quarto's `Render` button even for the Rmarkdown file; it seems pressing it also runs quarto rather than the original Rmarkdown chain.

### Describe the behavior you expected

I would expect that UI offers the `Knit`button for `.Rmd` files when the corresponding tab is active. Alternatively, the `Render`button should allow processing with Rmarkdown instead of Quarto.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@cmarquardt Than you for raising this!

@dragonstyle I'm not sure—is this intentional behavior or a bug?

**Comment by stevenbedrick** [comment]
I am encountering this exact problem, and am glad to have found this issue as I have just spent rather a long time trying to figure out why my document wasn't rendering the way I expected it to. Does anybody know if there is a way to change this behavior? I.e., get my one-off RMarkdown document to render using Knitr as usual? I was able to work around the issue by manually calling `rmarkdown::render()` in the console but obviously that is not an ideal solution.
